src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBig;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.image {
+  display: flex;
+  width: 19px;
+  height: 19px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  margin-left: 26px;
+
+  padding: 1px;
+}
+
+.form {
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background-color: white;
+  border: 1px solid #e1e7ed;
+
+  align-items: center;
+}
+
+.input {
+  font-size: 16px;
+  /* stylelint-disable-next-line max-line-length */
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Avenir";
+  font-weight: 300;
+  height: 30px;
+  width: 100%;
+  border: none;
+  margin-left: 61px;
+  outline: none;
+
+  padding-left: 1px;
+}
+
+.form--mini {
+  height: 42px;
+}
+
+.image--mini {
+  width: 11px;
+  height: 11px;
+  background-image: url(images/Vector.png);
+  margin-left: 12px;
+}
```

just change background size for image, u dont need to download another
```suggestion
.image--mini {
  width: 11px;
  height: 11px;
  background-size: 11px;
  background-image: url(images/Search.svg);
  margin-left: 12px;
}
```